### PR TITLE
Mirador 3 IIIF page view

### PR DESCRIPTION
**Pull request by rwd**

### CODE REVIEW

File: pages/iiif/index.vue [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  };
+</script>
+
+<style lang="scss" scoped>
+  #viewer {
+    width: 100%;
+    height: 100%;
+    position: fixed;
+  }
+</style>
```
[the remark this answers is not in the document]
Comment (rwd) [reply pages/iiif/index.vue]: Apparently not. Removed.

File: tests/features/pages/record-page.feature [hunk 2]
```diff
@@ -66,10 +66,10 @@ Feature: Record page
 
   Scenario: IIIF Image viewer
     When I open a `record page with a IIIF Image`
-    Then I see the `IIIF Image viewer`
+    Then I don't have the `IIIF viewer`
```
Comment (lbiedinger) [inline tests/features/pages/record-page.feature]: this expectation is wrong, as it'll show the IIIF viewer.

File: pages/record/_.vue [hunk 3]
```diff
@@ -230,7 +231,10 @@
         return langMapValueForLocale(this.description, this.$i18n.locale);
       },
       isRichMedia() {
-        return isRichMedia(this.selectedMedia);
+        return isRichMedia(this.selectedMedia, {
+          iiif: Number(process.env.ENABLE_IIIF_MEDIA),
+          ssl: !Number(process.env.DISABLE_REDIRECT_SSL)
```
[the remark this answers is not in the document]
Comment (rwd) [reply pages/record/_.vue]: Nowhere any longer and it's been removed.

File: tests/features/pages/record-page.feature (hunk 2 above)
Comment (rwd) [reply tests/features/pages/record-page.feature]: Indeed. It's been corrected.

**Pull request merged**